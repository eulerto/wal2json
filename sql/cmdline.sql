\set VERBOSITY terse

-- predictability
SET synchronous_commit = on;

SELECT 'init' FROM pg_create_logical_replication_slot('regression_slot', 'wal2json');

-- Unknown option
SELECT data FROM pg_logical_slot_get_changes('regression_slot', NULL, NULL,
	'nosuchopt', '42');

-- Regexp error
SELECT data FROM pg_logical_slot_get_changes('regression_slot', NULL, NULL,
	'include-table', '~(');

SELECT 'stop' FROM pg_drop_replication_slot('regression_slot');
