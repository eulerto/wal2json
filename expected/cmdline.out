\set VERBOSITY terse
-- predictability
SET synchronous_commit = on;
SELECT 'init' FROM pg_create_logical_replication_slot('regression_slot', 'wal2json');
 ?column? 
----------
 init
(1 row)

SELECT data FROM pg_logical_slot_get_changes('regression_slot', NULL, NULL, 'nosuchopt', '42');
ERROR:  option "nosuchopt" = "42" is unknown
-- Regexp error
SELECT data FROM pg_logical_slot_get_changes('regression_slot', NULL, NULL,
	'include-table', '~(');
ERROR:  invalid regular expression "(": parentheses () not balanced
SELECT 'stop' FROM pg_drop_replication_slot('regression_slot');
 ?column? 
----------
 stop
(1 row)

